packages/smart-accounts/tests/coinbase-account.test.ts

```diff
@@ -102,11 +109,15 @@ describe("Viem AA - Coinbase Smart Account", () => {
       owners: [owner],
     });
 
+    // Create a client with the account for gas manager hooks
+    const clientWithAccount = { ...client, account };
+
+    // Create a bundler client that uses the Alchemy gas manager
     const bundlerClient = createBundlerClient({
       account,
       chain: local070Instance.chain,
       transport: custom(client),
-      ...alchemyGasManagerHooks("test-policy"),
+      ...alchemyGasManagerHooks("test-policy", { client: clientWithAccount }),
```

Comment:
This doesn't actually verify that the prepared user op has the paymaster and it's related fields filled correctly - could we add assertions for that?